lexicant.html

```diff
@@ -68,7 +72,7 @@ <h1>Lexicant game</h1>
     function undo() {
       updateLettersState("pop")
       updateLettersState("pop")
-      url.searchParams.set("sequence", letters.innerHTML)
+      url.searchParams.set("sequence", letters)
```

Comment:
Since letters has moved from being an element to being a string, we need to update this.